src/xcode/ENA/ENA/Resources/Localization/de.lproj/privacy-policy.html

```diff
@@ -1,749 +1,330 @@
+		<p>Rechtsgrundlage dieser Verarbeitung Ihrer Zugriffsdaten und Gesundheitsdaten (Zufalls-IDs, Testergebnis, TAN und ggf. TeleTAN) ist Ihre Einwilligung für die Funktion „Testergebnis teilen“.</p>
+		<h3>d. Informatorische Nutzung der App</h3>
+		<p>Soweit Sie die App nur informatorisch nutzen, also keine der oben genannten Funktionen der App verwenden und keine Daten eingeben, findet die Verarbeitung ausschließlich lokal auf Ihrem Smartphone statt und es fallen keine personenbezogenen Daten an. In der App verlinkte Webseiten z.B.: <u>www.bundesregierung.de</u> werden im Standard-Browser Ihres Smartphones geöffnet und angezeigt. Welche Daten dabei verarbeitet werden hängt von dem genutzten Browser, dessen Konfiguration sowie der Datenverarbeitungspraxis der aufgerufenen Webseite ab.</p>
+		<h2>7. Welche Berechtigungen und Funktionen benötigt die App?</h2>
+		<p>Die App benötigt Zugriff auf verschiedene Funktionen und Schnittstellen Ihres Smartphones. Dazu ist es erforderlich, dass Sie der App bestimmte Berechtigungen erteilen. Die Berechtigungen sind von den verschiedenen Herstellern unterschiedlich programmiert. So können z. B. Einzelberechtigungen zu Berechtigungskategorien zusammengefasst sein, wobei Sie der Berechtigungskategorie nur insgesamt zustimmen können. Bitte beachten Sie, dass Sie im Falle der Ablehnung eines Zugriffs durch die App keine oder nur wenige Funktionen der App nutzen können.</p>
+
+		<h3>a. Technische Voraussetzungen (alle Smartphones)</h3>
+		<ul>
+			<li>Internet</li>
+		</ul>
+		<p>Die App benötigt für die Funktionen Risiko-Ermittlung, Testergebnisse erhalten und Testergebnis übermitteln eine Internetverbindung, um mit den Serversystemen der App kommunizieren zu können.</p>
+		<ul>
+			<li>Bluetooth</li>
+		</ul>
+		<p>Die Bluetooth-Schnittstelle Ihres Smartphones muss aktiviert sein, damit Ihr Smartphone Zufalls-IDs von anderen Smartphones erfassen und im Kontaktprotokoll des Geräts speichern kann.</p>
+		<ul>
+			<li>Kamera</li>
+		</ul>
+		<p>Ihr Smartphone benötigt eine Kamera, um damit einen QR-Code im Rahmen der Testregistrierung scannen können.</p>
+		<ul>
+			<li>Hintergrundbetrieb</li>
+		</ul>
+		<p>Die App nutzt den Hintergrundbetrieb (also wenn Sie die App nicht gerade aktiv nutzen), um Ihr Risiko automatisch zu ermitteln und den Status eines registrierten Tests abfragen zu können. Wenn Sie den Hintergrundbetrieb im Betriebssystem Ihres Smartphones deaktivieren, müssen Sie alle Aktionen in der App selbst starten.</p>
```

Comment:
this and a few other changes result in a different layout, where the textblock itself would not be aligned with the bullet point. Is this intended?

Reply:
Updated this now, changed from:
`
    <ul>
        <li>qwer</li>
    </ul>
    <p>asdf</p>
`
to
`
    <ul>
        <li>qwer<br/>
        asdf</li>
    </ul>
`